<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPotr</class>
 <widget class="QWidget" name="FPotr">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список потребителей</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/group.png</normaloff>:/res/1/group.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnInsert">
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Create.png</normaloff>:/res/Create.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Erase.png</normaloff>:/res/Erase.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/OK.png</normaloff>:/res/OK.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Отменить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/No.png</normaloff>:/res/No.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Sync.png</normaloff>:/res/Sync.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVib">
        <property name="text">
         <string>Выбрать</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/Red mark.png</normaloff>:/res/Red mark.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>395</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
